src/components/audio-zone.js

```diff
@@ -42,17 +49,35 @@ AFRAME.registerComponent("audio-zone", {
     this.enableDebug(this.data.debuggable && window.APP.store.state.preferences.showAudioDebugPanel);
   },
 
+  tick: (() => {
+    const quat = new THREE.Quaternion();
+    return function() {
+      this.el.object3D.quaternion.set(0, 0, 0, 1);
+      this.el.object3D.quaternion.multiply(this.el.object3D.parent.getWorldQuaternion(quat).inverse());
```

Comment:
I'm not so familiar with math, but I guess 

```
quaternion.set(0,0,0,1)
quaternion.multiply(other);
```

can be rewritten to

```
quaternion.copy(other);
```

?

Reply:
Yes, that's better.